extension/src/openvic-extension/utility/UITools.cpp

```diff
@@ -66,7 +71,7 @@ GUI::Position const* UITools::get_gui_position(String const& gui_scene, String c
 	return position;
 }
 
-static Array get_events_from_shortcut_key(String const& key) {
+Array get_events_from_shortcut_key(String const& key) {
```

Comment:
Why not make `get_events_from_shortcut_key` and `try_create_shortcut_action_for_button` static functions, local to `UITools.cpp`?

Reply:
Because there is no reason for the function to be accessible to any other translation unit.

Reply:
Yeah that's what I mean, make them have the `static` prefix in `UITools.cpp` so they definitely stay just in this translation unit